crates/swc_ecma_transforms_typescript/src/inline_enum.rs

```diff
@@ -0,0 +1,135 @@
+use std::{cell::RefCell, rc::Rc};
+
+use serde::{Deserialize, Serialize};
+use swc_atoms::JsWord;
+use swc_common::collections::AHashMap;
+use swc_ecma_ast::{
+    AssignExpr, ComputedPropName, Expr, Id, Lit, MemberExpr, MemberProp, Module, Script, Str, Tpl,
+};
+use swc_ecma_utils::ident::IdentLike;
+use swc_ecma_visit::{as_folder, noop_visit_mut_type, Fold, VisitMut, VisitMutWith};
+
+pub type TSEnumLit = Rc<RefCell<AHashMap<Id, AHashMap<JsWord, Lit>>>>;
```

Comment:
Can you change this to a struct which hides the actual type?
The field can be like

```rust
pub(crate) values: Rc<RefCell<AHashMap<Id, AHashMap<JsWord, Lit>>>>
```